service-base-quarkus/src/test/resources/application.yaml

```diff
@@ -0,0 +1,64 @@
+hono:
+  app:
+    maxInstances: 1
+    startupTimeout: 60
+  server:
+    bindAddress: "10.2.0.1"
+    insecurePort: 11001
+    insecurePortBindAddress: "10.2.0.2"
+    insecurePortEnabled: true
+    nativeTlsRequired: true
+    port: 11000
+  amqp:
+    corsAllowedOrigin: "client.eclipse.org"
+    deviceIdPattern: "[a-z]+"
+    eventLoopBlockedCheckTimeout: 12000
+    maxPayloadSize: 128000
+    networkDebugLoggingEnabled: true
+    receiverLinkCredit: 1000
+    sendTimeOut: 5100
+    tenantIdPattern: "[A-Z]+"
+    waitForDownstreamConnectionEnabled: true
+  command:
+    addressRewriteRule: "([a-z_]+)/([\\w-]+) test-vhost/$1/$2"
+    amqpHostname: "command.hono.eclipseprojects.io"
+    certPath: "/etc/cert.pem"
+    connectTimeout: 1234
+    credentialsPath: "/etc/creds"
+    flowLatency: 321
+    host: "hono.eclipseprojects.io"
+    idleTimeout: 44444
+    initialCredits: 55
+    keyPath: "/etc/key.pem"
+    linkEstablishmentTimeout: 1111
+    maxFrameSize: 32000
+    maxMessageSize: 64000
+    maxSessionFrames: 30
+    minMaxMessageSize: 65000
+    name: "client"
+    password: "secret"
+    pathSeparator: "-"
```

Comment:
`GenericOptions#pathSeparator` is documented as `character separating the segments of target addresses.`.
Using `-` here seems strange to me - I don't see how this would work.
In general, it seems to me this pathSeparator isn't actually used in non-test code. So how about removing it altogether?

Reply:
In this test case we only verify the correct binding of properties. The particular value of the property is chosen arbitrarily.
Removing the pathSeparator altogether seems reasonable to me but should be done in a separate issue FMPOV.